File: Android Basic/Android-First-Game-Java-Edition-/java/com/example/androidfirstjavagame/TitleScreen.java
package com.example.androidfirstjavagame;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class TitleScreen extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_title_screen);

        /*
                Button startB = (Button) findViewById(R.id.StartButton);
                startB.setText("HELLO");
         */
    }

    public void goGameScreen(View view){
        Intent gameScreen = new Intent(this, GameScreen.class);
        startActivity(gameScreen);

        /*
        Tak zmieniamy aktywnosc
         */
    }



}

File: Android Basic/Android-First-Kotlin-Game/java/com/example/myfirstandroidkotlingame/TitleScreen.kt
package com.example.myfirstandroidkotlingame

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_title_screen.*

class TitleScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_title_screen)

        StartButton.setOnClickListener {
            val Intent = Intent(this, GameScreen::class.java)
            startActivity(Intent)
        }


    }


}